packages/@webex/plugin-cc/src/services/core/WebSocket/connection-service.ts

```diff
@@ -1,5 +1,5 @@
 import {EventEmitter} from 'events';
-import {WebSocketManager} from './WebSocketManager';
+import {WebSocketManager} from './WebsocketManager';
```

Comment:
For some reason I'm not seeing a change in the file name... Is that just me? Also, do we want to change the class name also to WebsocketManager?

Reply:
The build is failing because of this.

Reply:
Build will pass now, this happened due to missing the last commit push